test/script/docker/run_tests.sh

```diff
@@ -24,17 +24,17 @@ else
   CMD='npm test'
 fi
 
-docker build --pull --force-rm \
-  --build-arg NODE_VERSION=$1 \
-  -t apm-agent-nodejs:node-${1} \
-  ./test
-
 NODE_VERSION=${1} docker-compose -f ./test/docker-compose.yml -f ./test/docker-compose.ci.yml run \
-  -e NODE_VERSION=${NODE_VERSION} -e TAV=${TAV_VERSIONS} -e CI=true \
+  -e NODE_VERSION=${NODE_VERSION} \
+  -e TAV=${TAV_VERSIONS} \
+  -e JUNIT=${JUNIT} \
+  -e CI=true \
+  -e ELASTIC_APM_ASYNC_HOOKS=false \
```

Comment:
Why are we disabling async_hooks here?

Reply:
C&P from Travis file

Reply:
Ah, we _should_ actually be testing both states. On Travis, some jobs include that and some don't. Specifically, every version of Node.js from 8 and above should test both states.

Reply:
The current setup tests with it enabled, so should we maybe just remove this line for now and land, then we can figure out a matrix test to cover both states later?

Reply:
yep, we can make it on the matrix test